modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/ObjectRelationshipInheritanceCheckbox.tsx

```diff
@@ -73,6 +50,17 @@ export function ObjectRelationshipInheritanceCheckbox({
 					)}
 				</ClayPopover>
 			</div>
+
+			{!!values.edge && (
+				<ClayAlert
+					displayType="info"
+					title={`${Liferay.Language.get('info')}:`}
+				>
+					{Liferay.Language.get(
+						'when-enabled,-permissions-are-inherited,-all-api-endpoints-are-grouped-under-the-parent,-and-the-relationship-field-is-always-mandatory'
+					)}
+				</ClayAlert>
+			)}
```

Comment:
I will check with the design team, but because the story for the model builder sidebar will still be developed, we agreed that this can move forward this way.